cpp/src/parquet/column_reader.cc

```diff
@@ -275,6 +275,9 @@ class SerializedPageReader : public PageReader {
   }
 
   // Implement the PageReader interface
+  //
+  // SerializedPageReader reuse same decryption buffer and decompression buffer,
```

Comment:
```suggestion
  // SerializedPageReader reuses same decryption buffer and decompression buffer,
```

Reply:
Should we make it clear when and which buffer will be reused? For example,
- If compression is enabled, decompression_buffer_ will be reused.
- If compression is not enabled but encryption is enabled, decryption_buffer_ will be reused.
- Neither encryption nor compression is enabled, the buffer is obtained directly from reading the stream.

Reply:
Seems that `decryption_buffer_` and `decompression_buffer_` is not conflict, and finally returned value might be one of them